postgresql/Dockerfile

```diff
@@ -26,7 +26,7 @@ LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 ENV POSTGRESQL_MAX_CONNECTIONS 200
 
 USER root
-
+RUN apt-get update && apt-get -y upgrade
```

Comment:
Need to also bump the version.txt due to this change, unless the existing version has not yet been released.

Reply:
not needed, I guess, its the just apt-update+upgrade on debian..

Reply:
What I mean is that if we don't update version.txt, then if we merge this and build it, the push will fail because `quay.io/astronomer/ap-postgresql:11.15.0` already exists. Because of this, we need to bump the version.txt to `11.15.0-1` which will produce a new image `quay.io/astronomer/ap-postgresql:11.15.0-1`